mangaki/mangaki/wrappers/anilist.py

```diff
@@ -247,8 +289,20 @@ def _request(self,
             query_params = {}
 
         r = self._session.get(urljoin(self.BASE_URL, datapage.format(**params)), params=query_params)
+        if r.status_code == 404:
```

Comment:
Indeed, bad call (for example, if AniList were to change its API routes one day). Fixed ! 🙇